app/views/licences/tabs/set-up.njk

```diff
@@ -10,135 +10,215 @@
   {% endfor %}
 {% endmacro %}
 
-{% set chargeVersionRows = [] %}
-
-{% set chargeVersionHead = [
-  {
-    text: "Start date"
-  },
-  {
-    text: "End date"
-  },
-  {
-    text: "Reason"
-  },
-  {
-    text: "Status"
-  },
-  {
-    text: "Action"
-  }
-] %}
-
-{% for chargeVersion in  chargeInformation %}
-  {% set chargeVersionRows = (chargeVersionRows.push([
+<h2 class="govuk-heading-l">Licence set up</h2>
+
+{% if enableRequirementsForReturns %}
```

Comment:
It's only the buttons we need to put behind the feature flag now. The work on cleaning up the requirements during import is now done enough that it is fine to display them to the users. In fact, the sooner we can the sooner they can confirm if they are as expected.

Reply:
Okay will change. 

Will update the tests